src/opm/output/eclipse/Summary.cpp

```diff
@@ -1162,41 +1153,29 @@ inline std::vector<Opm::Well> find_wells( const Opm::Schedule& schedule,
         }
 
         return wells;
+
     }
 
-    return {};
+    default:
```

Comment:
Preferably not.  Default clauses for switches on enums are more trouble than they're worth.

Reply:
I'm not sure what you mean by this?

Reply:
>> Default clauses for switches on enums are more trouble than they're worth.
>
> I'm not sure what you mean by this?

By adding a `default` clause to a switch on a strong enum you remove the compiler's ability to tell you if there's a case you've forgotten.  This isn't so much an issue when you're writing the code in the first place, but does become a potential error during subsequent maintenance.  I prefer having compile-time warnings/errors to having to do expensive runtime debugging.

Reply:
Addressed.